concent_api/core/templatetags/admin_tags.py

```diff
@@ -0,0 +1,52 @@
+import os
+from django import template
+from constance import config
+from ..models import Subtask
```

Comment:
I personally would rather see `from core.models import Subtask` for readability, but I don't know if there are different reason to use this style of imports. @rwrzesien @dybi ?

Reply:
Done.